src/api/index.js

```diff
@@ -18,14 +18,16 @@ const config = new Conf();
  * Checks if the plugin/package exists on npm
  *
  * @param {String} plugin - The name of the plugin/package
- * @return {Promise} - Throws an error if the plugin/package doesn't exist
+ * @return {Promise} - True if the plugin is found
  */
 const checkOnNpm = plugin => new Promise(resolve => {
-  npmName(plugin).then(available => {
-    if (available) {
-      throw new Error(ERR_MODULE_NOT_FOUND);
+  npmName(plugin).then(notFound => {
```

Comment:
it should be fairly simple to write our own to avoid this confusion.
This won't be the last time I have re-read the code 3-4 times to make sure I don't get confused...

skipping this for now